docs/conf.py

```diff
@@ -0,0 +1,204 @@
+# -*- coding: utf-8 -*-
+#
+# hipCUB documentation build configuration file, created by
+# sphinx-quickstart on Mon Jan  8 16:34:42 2018.
+#
+# This file is execfile()d with the current directory set to its
+# containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+import os
+import sys
+import subprocess
+
+read_the_docs_build = os.environ.get('READTHEDOCS', None) == 'True'
+
+# -- General configuration ------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+#
+# needs_sphinx = '1.0'
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = ['sphinx.ext.mathjax', 'breathe', 'exhale']
+breathe_projects = { "hipCUB": "./docBin/xml" }
+breathe_default_project = "hipCUB"
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+# The suffix(es) of source filenames.
+# You can specify multiple suffix as a list of string:
+#
+# source_suffix = ['.rst', '.md']
+source_suffix = '.rst'
+
+# The master toctree document.
+master_doc = 'index'
+
+# General information about the project.
+project = u'hipCUB'
+copyright = u'2010-2011, Duane Merrill; 2011-2018, NVIDIA CORPORATION; Modifications Copyright 2019 Advanced Micro Devices'
```

Comment:
I would check this line

Reply:
That's copied from [LICENSE.txt](https://github.com/ROCmSoftwarePlatform/hipCUB/blob/1bc92d7af3e7846cfa55ed82af6fb52ac8a4122c/LICENSE.txt)

Reply:
Copyright 2019-2021 Advanced Micro Devices
If you have a moment, the license file should be updated as well.